source/configure/configuration-settings.rst

```diff
@@ -1928,6 +1928,19 @@ Enable Developer Mode
 | This feature's ``config.json`` setting is ``"EnableDeveloper": false`` with options ``true`` and ``false``. |
 +-------------------------------------------------------------------------------------------------------------+
 
+Enable Client Performance Debugging
```

Comment:
Is a Telemetry page update needed as well for this new config setting?

Reply:
Added. I also fixed a random x in those docs that seemed like a mistake.